ui/jobs/jobs.css

```diff
@@ -674,6 +686,42 @@
   background-color: rgb(60, 80, 200);
 }
 
+.brd-color-causticbite {
+  background-color: rgb(182, 68, 235);
+}
+
+.brd-color-stormbite {
+  background-color: rgb(72, 117, 202);
+}
+
+.brd-color-song {
+  background-color: rgb(180, 200, 220);
+}
+
+.brd-color-song.minuet {
+  background-color: rgb(100, 150, 100);
+}
+
+.brd-color-song.minuet.full {
+  background-color: rgb(250, 150, 0);
+}
+
+.brd-color-song.ballad {
+  background-color: rgb(143, 90, 143);
+}
+
+.brd-color-song.paeon {
+  background-color: rgb(207, 205, 52);
+}
+
+.brd-color-soulvoice {
+  background-color: rgb(130, 200, 0);
```

Comment:
I feel like the soulvoice and the minuet colors both are green and kind of similar.  I also think the high color being yellowish looks a lot like the paeon color.  Maybe there can be more differentiation?  I like the songs being different colors, but maybe then soul voice doesn't need as much color normally?

What do you think about soul voice being white-ish / uncolored and then the high color being red-ish or more orange-ish rather than yellow-ish?

Reply:
I think so. I mostly pick a color on its icon, but sometimes there is too many similar color.